.github/workflows/windows.yml

```diff
@@ -654,7 +654,10 @@ jobs:
         run: |
           export TFLAGS='-j14 !TFTP !MQTT !WebSockets !SMTP ~FTP ${{ matrix.tflags }}'
           if [[ '${{ matrix.config }}' = *'-DUSE_WIN32_IDN=ON'* ]]; then
-            TFLAGS+=' ~165 ~1448 ~2046 ~2047'
+            TFLAGS+=' ~165'
```

Comment:
I think it's better to remove the if, and put the relevant ignore tests in all places that have unicode in matrix configuration.

Reply:
For failures with a specific or unique combinations, the per-job exception remains an option. It also works for those with no analysis, just to silence them.

But, once we discover a connection between option and failure, the rule based method is useful to document it and can help fixing the problem.